sdk/src/main/java/io/radar/sdk/RadarLocationReceiver.kt

```diff
@@ -99,11 +99,11 @@ class RadarLocationReceiver : BroadcastReceiver() {
     @SuppressLint("MissingPermission")
     override fun onReceive(context: Context, intent: Intent) {
         if (!Radar.initialized) {
-            // Radar must be initialized before using Radar.logger
```

Comment:
I think it’s important for this comment to be here because it’s not obvious that initialize needs to be called before using `Radar.logger`. Accidentally removing the initialize call at some point in the future would lead to rare crashes that might not be caught during QA.